CausalEstimate/vis/plotting.py

```
@@ -365,3 +374,120 @@ def plot_calibration_comparison(
+
+    Args:
+        balance_table: Output of CausalEstimate.diagnostics.compute_balance_table.
+        threshold: |SMD| bound drawn as the balance reference line.
+        fig, ax, figsize: As in plot_hist_by_groups.
+
+    Returns:
+        (fig, ax)
+    """
+    table = balance_table[[SMD_UNWEIGHTED_COL, SMD_WEIGHTED_COL]].abs()
+    if table.isna().any(axis=None):
+        warnings.warn(
```

Ruff (0.16.2)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)